portfolio/src/main/webapp/style.css

```diff
@@ -137,19 +137,26 @@ display: none;
   border-style: groove;
   padding: 3px 7px 3px 3px;
   margin-top: 0;
+  margin-right: 10px;
 }
 
 .name{
-    margin-bottom: 0;
+  margin-bottom: 0;
+}
+
+.remove {
+  float:left;
+  margin-top: 25px;
+  margin-right: 7px;
 }
 
 #messages-container {
-    background-color: #fafafa;
-    padding-left: 20px;
-    padding-top: 1px;
-    margin-top: 10px;
-    height: 500px;
-    overflow: scroll;
+  background-color: #fafafa;
+  padding-left: 20px;
+  padding-right: 20px;
```

Comment:
Protip for the future, no change needed here.

- use px for borders and shaddows
- use rem for all other types (particularly padding, alignment, margin)
- use Calc(...) with vh and vw for positioning relative to screen height.

https://www.sitepoint.com/understanding-and-using-rem-units-in-css/